src/Content/Version.php

```diff
@@ -195,6 +187,41 @@ public function id(): VersionId
 		return $this->id;
 	}
 
+	/**
+	 * Checks if the version is the latest version
+	 */
+	public function isLatest(): bool
+	{
+		return $this->id->is(VersionId::latest());
```

Comment:
`Suggestion`: Since `VersionId::is()` is casting it to a string anyways

```suggestion
		return $this->id->is('latest');
```